src/main.js

```diff
@@ -79,6 +82,7 @@ function deleteFromArray(event) {
       savedIdeas.splice(savedIdeas.indexOf(savedIdeas[i]));
     }
   }
+  saveIdeaToStorage()
 }
```

Comment:
Updates the savedIdeas array and saves this new array to localStorage, overwriting what is currently saved there.